docs/on-prem.rst

```diff
@@ -61,6 +65,7 @@ If you are using Docker:
 Gitpod
 ~~~~~~
 
+This demo is enabled to run with Gitpod, but support for the Gitpod workflow is best effort from the `community <https://github.com/confluentinc/cp-demo>`__.
```

Comment:
I'm not sure, but would it be best to point users directly to the GH Issues page?
https://github.com/confluentinc/cp-demo/issues

Reply:
Ack, changed to issues